.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.4/apache-maven-3.8.4-bin.zip
```

Comment:
PR does not have the latest Maven of 3.8.6 but rather 3.8.4... which further convinces me not to do this.